initfiles/etc/DIR_NAME/environment.conf.in

```diff
@@ -66,3 +66,13 @@ additionalPlugins=python2
 useDropZoneRestriction=true
 # If set, will force matching local file paths to become remote reads, e.g:
 #forceRemotePattern=/var/lib/HPCCSystems/hpcc-data/eclagent/*
+
+# Dafilesrv: default client side connection settings (NB: 0 = disable/use system defaults)
+#dafsConnectTimeoutSeconds=100
+#dafsConnectRetries=2
+#dafsMaxReceiveTimeSeconds=0
+
+# Dafilesrv: set to change number of seconds before retrying an unresponsive dafilesrv connection (default 10 seconds)
+# NB: for now this only applies to the last cached server 
+#dafsConnectFailRetrySeconds=10
+
```

Comment:
If you add to/change the environment.conf file you should also update:
initfiles/etc/DIR_NAME/environment.xml.in (in community and LN branches)
testing/regress/environment.xml.in
docs/*/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml
initfiles/componentfiles/configschema/xsd/envsettings.xsd

Reply:
Agreed, but environment.xml not changed so esting/regress/environment.xml.in and LN changes not required.

Will update HPCCSystemAdministratorsGuide.xml and envsettings.xsd

Reply:
The top of the xml matches the conf - so I think its good to have them continue to match.

Reply:
hm, yes agreed, though not sure anything uses them.
Nevertheless I'll update.

Reply:
hmm there's loads of settings that haven't been put in there, making me doubt it's usefulness all the more.

Reply:
Given that only some duplicated into <EnvSettings> that are in environment.conf, and there a bunch of others which are solely in <EnvSettings> (not in environment.conf), I won't add more duplicates into environment.xml

I think probably want needs to happen, is that the duplicates, that afaics are never read fromenvironment.xml, needs clearing up and removing.
Needs another JIRA to look at that though.

Reply:
ok, sure.